src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        Color color = Color.values()[index];
+
+        return color;
```

Comment:
**Don’t create temporary variables if you can directly return value from the method**
Bad example:
```
public Color getRandomColor() {
        int index = random.nextInt(Color.values().length);
        Color color = Color.values()[index];
        return color;
}
```
Improved example:
```
public Color getRandomColor() {
        int index = random.nextInt(Color.values().length);
        return Color.values()[index];
}
```